Bigtable/src/Instance.php

```diff
@@ -159,7 +159,7 @@ public function name()
      * Example:
      * ```
      * $info = $instance->info();
-     * echo $info['nodeCount'];
+     * echo $info['serveNodes'];
```

Comment:
`serveNodes` exists on a [cluster](https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#cluster). I believe we need to showcase a property of an [instance](https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#instance)?